src/client.js

```diff
@@ -0,0 +1,93 @@
+const formatsByCoinType = require('@ensdomains/address-encoder').formatsByCoinType;
+const Command = require('commander').Command;
+const ethers =  require('ethers');
+const OffchainResolver_abi = require('@ensdomains/offchain-resolver-contracts/artifacts/contracts/OffchainResolver.sol/OffchainResolver.json').abi;
+const IResolverService_abi = require('@ensdomains/offchain-resolver-contracts/artifacts/contracts/OffchainResolver.sol/IResolverService.json').abi;
+const Resolver_abi = require('@ensdomains/ens-contracts/artifacts/contracts/resolvers/Resolver.sol/Resolver.json').abi;
+
+const IResolverService = new ethers.utils.Interface(IResolverService_abi);
+
+function getDnsName(name) {
```

Comment:
Can you import mafintosh/dns and use `name.encode`, or at least copy the code here?